### PR TITLE
Patient resource patient identifier section

### PR DESCRIPTION
**Pull request by dino-saurabh**
1. Added examples
2. corrected payload
3. added annotations

### CODE REVIEW

**Comment by dino-saurabh** [comment]
@Akayeshmantha please review !!

File: source/includes/Patient/patients.md [hunk 1]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
+  .addHeader("Cookie", "JSESSIONID=33B84B3BEA8E81E9D22D5A722815E010")
+  .build();
+Response response = client.newCall(request).execute();
 
-```console
+```
+
+```java
+
+var myHeaders = new Headers();
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md]: Can we change myheader

File: source/includes/Patient/patients.md [hunk 2]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
+
+```
+
+> Success Response
+
+```response
+
+{
+    "results": [
+        {
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md:187]: Feels like this one is not algines well correct me if am wrong

File: source/includes/Patient/patients.md [hunk 3]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
 ```
 
+```java
+
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+MediaType mediaType = MediaType.parse("application/json");
+RequestBody body = RequestBody.create(mediaType, "{ \r\n    \"identifier\" : \"111:CLINIC1\",\r\n    \"identifierType\" : \"a5d38e09-efcb-4d91-a526-50ce1ba5011a\",\r\n    \"location\" : \"8d6c993e-c2cc-11de-8d13-0010c6dffd0f\",\r\n    \"preferred\" : true\r\n}\r\n");
+Request request = new Request.Builder()
+  .url("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/patient/e739808f-f166-42ae-aaf3-8b3e8fa13fda/identifier/")
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md]: Remove qa base server url

File: source/includes/Patient/patients.md [hunk 4]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
+var raw = JSON.stringify({"identifier":"111:CLINIC1","identifierType":"a5d38e09-efcb-4d91-a526-50ce1ba5011a","location":"8d6c993e-c2cc-11de-8d13-0010c6dffd0f","preferred":true});
+
+var requestOptions = {
+  method: 'POST',
+  headers: myHeaders,
+  body: raw,
+  redirect: 'follow'
+};
+
+fetch("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/patient/e739808f-f166-42ae-aaf3-8b3e8fa13fda/identifier/", requestOptions)
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md]: Ditto

File: source/includes/Patient/patients.md [hunk 5]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
 ```
 
+```java
+
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+MediaType mediaType = MediaType.parse("application/json");
+RequestBody body = RequestBody.create(mediaType, "{ \r\n    \"identifier\" : \"111:CLINIC2\"\r\n}\r\n");
+Request request = new Request.Builder()
+  .url("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/patient/e739808f-f166-42ae-aaf3-8b3e8fa13fda/identifier/beb6be1f-07a3-484c-a4ff-92fcb566ddde")
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md]: Ditto

File: source/includes/Patient/patients.md [hunk 6]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
+var raw = JSON.stringify({"identifier":"111:CLINIC2"});
+
+var requestOptions = {
+  method: 'POST',
+  headers: myHeaders,
+  body: raw,
+  redirect: 'follow'
+};
+
+fetch("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/patient/e739808f-f166-42ae-aaf3-8b3e8fa13fda/identifier/beb6be1f-07a3-484c-a4ff-92fcb566ddde", requestOptions)
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md]: Ditto

File: source/includes/Patient/patients.md [hunk 7]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
-* Delete or retire a target identifier subresource by its UUID. Returns a `404 Not Found` status if attribute not exists. 
-If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
+```java
+
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+MediaType mediaType = MediaType.parse("text/plain");
+RequestBody body = RequestBody.create(mediaType, "");
+Request request = new Request.Builder()
+  .url("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/patient/c7ea5ea9-bec7-4ad0-a803-0ef2dee8fca5/identifier/e5ce3659-9118-4912-8c2f-6d470c2c7940?purge=true")
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md]: Ditto

File: source/includes/Patient/patients.md [hunk 8]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
+myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+myHeaders.append("Cookie", "JSESSIONID=33B84B3BEA8E81E9D22D5A722815E010");
+
+var requestOptions = {
+  method: 'DELETE',
+  headers: myHeaders,
+  redirect: 'follow'
+};
+
+fetch("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/patient/c7ea5ea9-bec7-4ad0-a803-0ef2dee8fca5/identifier/e5ce3659-9118-4912-8c2f-6d470c2c7940?purge=true", requestOptions)
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md]: Ditto

File: source/includes/Patient/patients.md [hunk 9]
```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
...
+
+```
+
+```javascript
+
+var myHeaders = new Headers();
+myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+myHeaders.append("Cookie", "JSESSIONID=33B84B3BEA8E81E9D22D5A722815E010");
+
+var requestOptions = {
```
Comment (Akayeshmantha) [inline source/includes/Patient/patients.md:442]: Ditto

**Review by Akayeshmantha — CHANGES_REQUESTED** [review]
@LuGO0 looks good small changes and we can merge the PR

File: source/includes/Patient/patients.md (hunk 2 above)
Comment (dino-saurabh) [reply source/includes/Patient/patients.md:187]: ok @Akayeshmantha I will check once again for alingment

File: source/includes/Patient/patients.md (hunk 3 above)
Comment (dino-saurabh) [reply source/includes/Patient/patients.md]: yes @Akayeshmantha My bad I forgot to find and replace with the base server url doing it right away 👍

**Comment by Akayeshmantha** [comment]
LGTM

**Pull request merged**